pasture-core/src/gpu/gpu_point_buffer.rs

```diff
@@ -525,7 +525,7 @@ impl GpuPointBufferInterleaved {
         let gpu_buffer = self.buffer.as_ref().unwrap();
         wgpu_queue.write_buffer(&gpu_buffer, offset as wgpu::BufferAddress, bytes_to_write);
 
-        self.create_bind_group(wgpu_device);
+        // self.create_bind_group(wgpu_device);
```

Comment:
The commented code should be removed, or an additional comment should be added stating why it is left in here, but commented out